<?xml version="1.0" encoding="utf-8"?>

<launch>

  <arg name="rviz" default="true" />
  <node pkg="odom_based_3d_slam" type="pointcloudRegistration" name="pointcloudRegistration"  output="screen">
    <param name="turningRegTresh" type="double" value="9.3" />
    <param name="timeStampCompensation" type="int" value="50000000" />
    <param name="testVal" type="double" value="2.14" />
  </node>


  <!-- /home/youliang/rosbag/imu_office_large.bag  /imu/imu:=/imu_null"    imu_office_large_filter  mapping_bot_filtered  mapping_bot_21Oct19.bag -->
  <node name="player" pkg="rosbag" type="play" args=" -r 0.5 --clock  /home/youliang/rosbag/testFarm.bag /tf:=/tf_dev_null "/>
  
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find odom_based_3d_slam)/rviz/PointCloudRegistration.rviz" />
  </group>
  

</launch>
